redash/__init__.py

```diff
@@ -68,71 +60,13 @@ def create_redis_connection():
 
 
 setup_logging()
-redis_connection = create_redis_connection()
-
-mail = Mail()
-migrate = Migrate()
-mail.init_mail(settings.all_settings())
-statsd_client = StatsClient(host=settings.STATSD_HOST, port=settings.STATSD_PORT, prefix=settings.STATSD_PREFIX)
-limiter = Limiter(key_func=get_ipaddr, storage_uri=settings.LIMITER_STORAGE)
 
-import_query_runners(settings.QUERY_RUNNERS)
-import_destinations(settings.DESTINATIONS)
+redis_connection = LocalProxy(create_redis_connection)
 
-from redash.version_check import reset_new_version_status
-reset_new_version_status()
+mail = LocalProxy(Mail)
 
+migrate = LocalProxy(Migrate)
 
-class SlugConverter(BaseConverter):
-    def to_python(self, value):
-        # This is ay workaround for when we enable multi-org and some files are being called by the index rule:
-        # for path in settings.STATIC_ASSETS_PATHS:
-        #     full_path = safe_join(path, value)
-        #     if os.path.isfile(full_path):
-        #         raise ValidationError()
-
-        return value
-
-    def to_url(self, value):
-        return value
-
-
-def create_app():
```

Comment:
Can we import `create_app` to keep things that were importing it from `redash` working?

Reply:
Ah, yes, I meant to do that. Good idea.